nosetimer/plugin.py

```diff
@@ -60,7 +60,12 @@ def configure(self, options, config):
             self.timer_top_n = int(options.timer_top_n)
             self.timer_ok = self._parse_time(options.timer_ok)
             self.timer_warning = self._parse_time(options.timer_warning)
-            self.timer_no_color = options.timer_no_color
+
+            # Windows + nosetests does not support colors (even with colorama).
+            if os.name == 'nt':
```

Comment:
Can we make this into a custom argparser action that disables this option instead of an having if/else statement?